src/client.rs

```diff
@@ -820,27 +851,27 @@ mod tests {
     #[meilisearch_test]
     async fn test_get_keys(client: Client) {
         let keys = client.get_keys().await.unwrap();
-        assert!(keys.len() >= 2);
-        assert!(keys.iter().any(
-            |k| k.description != "Default Search API Key (Use it to search from the frontend)"
-        ));
-        assert!(keys.iter().any(
-            |k| k.description != "Default Admin API Key (Use it for all other operations. Caution! Do not use it on a public frontend)"
+        assert!(keys.results.len() >= 2);
+        assert!(keys.results.iter().any(|k| k.description
+            != Some("Default Search API Key (Use it to search from the frontend)".to_string())));
+        assert!(keys.results.iter().any(
+            |k| k.description != Some("Default Admin API Key (Use it for all other operations. Caution! Do not use it on a public frontend)".to_string())
         ));
     }
 
     #[meilisearch_test]
-    async fn test_delete_key(client: Client, description: String) {
-        let key = KeyBuilder::new(description);
+    async fn test_delete_key(client: Client) {
+        let mut key = KeyBuilder::new();
+        key.with_name("test_delete_key");
```

Comment:
```suggestion
    async fn test_delete_key(client: Client, name: String) {
        let mut key = KeyBuilder::new();
        key.with_name(&name);
```

We have a cool function to get a unique string.

Reply:
woops